### PR TITLE
API docs for proj membership and proj members

### PR DESCRIPTION
**Pull request by achakrabarti-smartsheet**
Includes separate sections for Project Membership and Project Member, along with all the needed endpoints.

### CODE REVIEW

File: source/includes/project-membership.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Project Membership
+
+##### Endpoint: `/api/v1/users/:id/project_memberships`
+##### Endpoint: `/api/v1/users/:id/project_members`
```
Comment (tithonium) [inline source/includes/project-membership.md]: this should be v1/projects/

File: source/includes/project-membership.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+This API call updates the project's membership list. JSON body should include a `relationships` field with `[user_id, membership_level]` tuples.
+
+NOTE: This updates the membership list. If you leave off a project, the user’s relationship to that project will be unchanged. If the user was already a member of a project, their membership will be changed to the level specified. If the user was not previously a member of a project, they will be added with the level specified.
+
+## Remove
+```
+DELETE /api/v1/projects/:id/project_members
+```
+This API call removes users from the project's membership list. JSON body should include a `relationships` field, which is an array of `[user_id, membership_level]` tuples, OR a `user_ids` field, which is an array of `[user_id, user_id, ...]`.
```
Comment (tithonium) [inline source/includes/project-membership.md]: I would say only document the user_ids field, as the other is potentially confusing and not as useful.

File: source/includes/project-membership.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+This API call updates the user's project memberships. JSON body should include a `relationships` field with `[project_id, membership_level]` tuples.
+
+NOTE: This updates the membership list. If you leave off a project, the user’s relationship to that project will be unchanged. If the user was already a member of a project, their membership will be changed to the level specified. If the user was not previously a member of a project, they will be added with the level specified.
+
+## Remove
+```
+DELETE /api/v1/users/:id/project_memberships
+```
+This API call removes the single user from the project's membership list. JSON body should include a `relationships` field, which is an array of `[project_id, membership_level]` tuples, OR a `project_ids` field, which is an array of `[project_id, project_id, ...]`.
```
Comment (tithonium) [inline source/includes/project-membership.md]: I would say only document the project_ids field, as the other is potentially confusing and not as useful.

**Review by tithonium — CHANGES_REQUESTED** [review]

File: source/includes/project-membership.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Project Membership
 
-##### Endpoint: `/api/v1/users/:id/project_memberships`
-##### Endpoint: `/api/v1/users/:id/project_members`
+##### Endpoint: `/api/v1/projects/:id/project_memberships`
+##### Endpoint: `/api/v1/projects/:id/project_members`
```
Comment (tithonium) [inline source/includes/project-membership.md:4]: Nope, just the second one. users have project_memberships, projects have project_members.

Comment (achakrabarti-smartsheet) [reply source/includes/project-membership.md:4]: Oh right... typo cuz the more detailed endpoints are literally in the following lines :/

**Review by tithonium — APPROVED** [review]

**Pull request merged**